### PR TITLE
fix: remove double kazana prefix for new config format and update tests

### PR DESCRIPTION
**Pull request by lucidNTR**
@gr2m @jo @lucychambers 

please review and merge asap
then update kazana

### CODE REVIEW

File: tests/integration/deprecated-42-test/expected.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
-  'port': '5000',
-  'db.rawData': 'my-raw-data',
-  'db.fooDb': 'bar',
+  'kazana.port': '5000',
```
Comment (jo) [inline tests/integration/deprecated-42-test/expected.js]: Isn't the depricated way without kazana prefix?

Comment (gr2m) [inline tests/integration/deprecated-42-test/expected.js]: yes

File: index.js [hunk 2]
```diff
@@ -33,9 +33,6 @@ function getConfig (main) {
     log.warn('deprecation', '.kazanarc: settings must be prefixed by one of: %s', rcNamespaceNames, {
       issue: 'https://github.com/eHealthAfrica/kazana-config/pull/42'
     })
-    rcConfig = {
-      kazana: rcConfig
-    }
```
Comment (gr2m) [inline index.js]: can you explain how this fixes the problem in your case? Because this will break other apps, we have to find another way

**Comment by gr2m** [comment]
merged via 9bb238a. I added `only` to the `bare` key, as it’s used to start the plugins in separate processes. It’s not a very clean solution though, because the `--bare` and `--only` arguments are somewhat out of scope of `kazana-config`. But that should fix the problem for now, great catch @lucidNTR 

I'll give it some more

**Pull request closed without merging**